templates/smcp-templates/v2.4/cr_2.4_default_template.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: maistra.io/v2
```

Comment:
This file is not used neither. `clean_members_50.sh` is used instead.

Reply:
Deleted too